<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upcomingEventsGeneralSettings</class>
 <widget class="QWidget" name="upcomingEventsGeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>60</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="filterLabel">
     <property name="text">
      <string>Filter events by date:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="filterComboBox">
     <item>
      <property name="text">
       <string>All events</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>This week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>This month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>This year</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="groupVenueCheckBox">
     <property name="text">
      <string>&amp;Group venue events</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
